### DISCUSSION TITLE
Arranging informal "office hours"

### DISCUSSION THREAD

**Discussion by ml-evs** [marda-alliance/metadata_extractors#6] [General]
For those of us who are interested in working on this project, it was suggested that we occasionally announce some informal times that we can work together on Jitsi (https://meet.jit.si/marda_extractors) and discuss/hack on how this project fits in with our own work.

We will try one of these next week on Thursday (8th December) with at least @PeterKraus and myself, from 14:00-16:00 UTC (15:00-17:00 CET, 09:00-11:00 EST). I will commit to staying for the full 2 hours but people can drop in and out as they please.

If anyone else wants to arrange something similar then please just drop a time/date in this thread!

**Comment by ml-evs** [comment]
Next office hours will be on [Jitsi](https://meet.jit.si/marda_extractors) on Tuesday January 24th  at 15:00-16:00 UTC (16:00-17:00 CET, 10:00-11:00 EST).

If you would like to come but can't make those times, please respond below here and we can arrange something else.

**Comment by ml-evs** [comment]
Next office hours will be on Jitsi (link above) on the 7th February at 15:00-16:00 UTC (16:00-17:00 CET)!

**Comment by ml-evs** [comment]
Next office hours will be 20th March at 15:00 UTC/16:00 CET!

**Comment by ml-evs** [comment]
Next office hours will be on [Jitsi](https://meet.jit.si/marda_extractors) on [Tuesday April 4th at 14:00 UTC](https://www.timeanddate.com/worldclock/fixedtime.html?iso=20230404T14) (16:00 CET, 10:00 EST).

**Reply by PeterKraus** [reply]
My personal agenda for these office hours:

- go through any outstanding issues, especially the low hanging fruits (example files for Filetypes, update yadg Extractor with a python usage example)
- work on PRs for defining `usage` and `installation` schemas
- set up LFS for example/test files in the registry

This should set us up nicely to allow to rework https://github.com/marda-alliance/metadata_extractors_api/pull/5 to be fully registry-based.

**Reply by PeterKraus** [reply]
I've made 2 PRs adding example/test files for `Filetypes` into the registry:

https://github.com/marda-alliance/metadata_extractors_registry/pull/30
https://github.com/marda-alliance/metadata_extractors_registry/pull/31

Those are good to go IMO.

I have also made a draft PR setting some aspects of the schema as required:

https://github.com/marda-alliance/metadata_extractors_schema/pull/21

This needs some work and discussions. I'd appreciate comments (@ml-evs @davidelbert)

Remaining on the todo are the `usage` and `installation` modifications for the schema. I'll try to get drafts up before end of next week. Once that's done, I'm going to hopefully also have a minted (pre-)version of `yadg`, which can then be reflected in the `registry`.

**Reply by PeterKraus** [reply]
The draft for `usage` and `installation` in the `Extractor` schema is here: https://github.com/marda-alliance/metadata_extractors_schema/pull/22; that's all from me for now (at least before Easter).

**Comment by ml-evs** [comment]
Next office hours will be Wednesday 26th April at 16:00 CST, hope to see you there!

**Comment by ml-evs** [comment]
Next office hours, 5th May at 14:00 CEST!

**Comment by ml-evs** [comment]
Next office hours are Wednesday 17th at 14:00 UTC/16:00 CEST!

**Comment by PeterKraus** [comment]
The next office hours are Thursday 22nd of June, 12:00 UTC / 14:00 CEST!

@ml-evs @eminizer @steffenbrinckmann

**Reply by ml-evs** [reply]
@edan-bainglass

**Reply by edan-bainglass** [reply]
Thank you Matthew :)

**Reply by ml-evs** [reply]
Forgot to say that the link for the meeting is in the top comment!

**Reply by edan-bainglass** [reply]
Jitsi link. Got it! Thanks again :) I'll try to make it this Thursday @ 2p my time.
